### ISSUE TITLE
Sensors unavailable after upgrade 

### ISSUE THREAD

**Issue by jes1417** [moralmunky/Home-Assistant-Mail-And-Packages#153]
Upgrading to 0.2.1-b6 has made all sensors disappear and I am getting errors in the log

```
Log Details (ERROR)

Logger: homeassistant.config_entries
Source: loader.py:273
First occurred: 11:39:20 AM (4 occurrences)
Last logged: 12:15:23 PM

Error setting up entry imap.gmail.com for sensorTraceback (most recent call last): File "/usr/src/homeassistant/homeassistant/config_entries.py", line 217, in async_setup hass, self File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 59, in async_setup_entry return await hass.data[DOMAIN].async_setup_entry(entry) File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 150, in async_setup_entry platform_type, File "/usr/src/homeassistant/homeassistant/setup.py", line 275, in async_prepare_setup_platform platform = integration.get_platform(domain) File "/usr/src/homeassistant/homeassistant/loader.py", line 273, in get_platform f"{self.pkg_path}.{platform_name}" File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module return _bootstrap._gcd_import(name[level:], package, level) File "<frozen importlib._bootstrap>", line 1006, in _gcd_import File "<frozen importlib._bootstrap>", line 983, in _find_and_load File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked File "<frozen importlib._bootstrap>", line 677, in _load_unlocked File "<frozen importlib._bootstrap_external>", line 724, in exec_module File "<frozen importlib._bootstrap_external>", line 860, in get_code File "<frozen importlib._bootstrap_external>", line 791, in source_to_code File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed File "/config/custom_components/mail_and_packages/sensor.py", line 699 deliveriesToday.append("Amazon Order") ^ IndentationError: expected an indented block


```

```
Log Details (ERROR)

Logger: homeassistant.config_entries
Source: helpers/entity_component.py:176
First occurred: 12:13:23 PM (3 occurrences)
Last logged: 12:15:23 PM

Error unloading entry imap.gmail.com for sensorTraceback (most recent call last): File "/usr/src/homeassistant/homeassistant/config_entries.py", line 304, in async_unload hass, self File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 64, in async_unload_entry return await hass.data[DOMAIN].async_unload_entry(entry) File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 176, in async_unload_entry raise ValueError("Config entry was never loaded!") ValueError: Config entry was never loaded!


```

I have tried reinstalling through hacs and reconfiguring the integration multiple times.

**Comment by firstof9** [comment]
Fixed in 0.2.1-b7-1

**Comment by jes1417** [comment]
👍 all good now
